docs/postgresql.md

```diff
@@ -0,0 +1,139 @@
+# `postgresql` Metrics Receiver
+
+The postgresql receiver can retrieve stats from your postgresql instance by connecting as a monitoring user.
+
+## Prerequisites
+
+The `postgresql` receiver defaults to connecting to a local postgresql server using a Unix socket and Unix authentication as the `postgres` user.
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your postgresql configuration.
+
+To configure a receiver for your postgresql metrics, specify the following fields:
+
+| Field                   | Default                         | Description |
```

Comment:
Can we add a column to indicate whether the config parameter is required or optional?